editoast/src/generated_data/mod.rs

```diff
@@ -87,19 +87,20 @@ pub trait GeneratedData {
 
 /// Refresh all the generated data of a given infra
 pub async fn refresh_all(
-    db_pool: Arc<DbConnectionPool>,
+    db_pool: Arc<DbConnectionPoolV2>,
     infra: i64,
     infra_cache: &InfraCache,
 ) -> Result<()> {
     // The other layers depend on track section layer.
     // We must wait until its completion before running the other requests in parallel
     TrackSectionLayer::refresh_pool(db_pool.clone(), infra, infra_cache).await?;
-    debug!("⚙️ Infra {infra}: track section layer is generated");
-    let mut conn = db_pool.get().await?;
+    debug!(infra_id = infra, "Infra: track section layer is generated");
```

Comment:
Maybe adding a `#[tracing::instrument(fields(infra_id)` instead to create a span would be more idiomatic? @woshilapin a confirmation ? 😅

Reply:
I just added `#[tracing::instrument...`